### PR TITLE
Add eslint/airbnb

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

**Review by davidthomas22 — DISMISSED** [review]
LGTM

File: server/routes/product.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-  const router = express.Router()
+module.exports = function product({ productService }) {
+  const router = express.Router();
   router.get('/product/:id', async (req, res) => {
-    const product = await productService.getEntry(req.params.id)
-    console.log('GET product')
-    product.fields.productOutcomes = richTextRenderer.documentToHtmlString(product.fields.productOutcomes)
-    product.fields.productRelatedProductsOrServices = richTextRenderer.documentToHtmlString(product.fields.productRelatedProductsOrServices)
-    product.fields.productExampleFeatures = richTextRenderer.documentToHtmlString(product.fields.productExampleFeatures)
+    const productEntry = await productService.getEntry(req.params.id);
```
Comment (jonhindle) [inline server/routes/product.js]: can we not just extract out fields: 
const { fields } = await productService.getEntry(req.params.id);

File: server/routes/product.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+    console.log('GET product');
+    const outcomes = richTextRenderer.documentToHtmlString(product.fields.productOutcomes);
+    const relatedProductsOrServices = richTextRenderer.documentToHtmlString(
+      product.fields.productRelatedProductsOrServices,
```
Comment (jonhindle) [inline server/routes/product.js]: `product` should be `productEntry` I think?

File: server/routes/product.js [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+    );
+    const exampleFeatures = richTextRenderer.documentToHtmlString(
+      product.fields.productExampleFeatures,
```
Comment (jonhindle) [inline server/routes/product.js]: Same again `product` should be `productEntry` I think?

File: server/routes/product.js (hunk 1 above)
Comment (psoleckimoj) [reply server/routes/product.js]: Done

File: server/routes/product.js (hunk 2 above)
Comment (psoleckimoj) [reply server/routes/product.js]: Yep

File: server/routes/product.js (hunk 3 above)
Comment (psoleckimoj) [reply server/routes/product.js]: Ye

File: server/routes/index.js [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+  const router = express.Router();
   router.get('/', async (req, res) => {
-    const rawProducts = await productService.getEntries('product')
-    const rawThemes = await productService.getEntries('themes')
-    const products = formatRawProductData(rawProducts)
-    const statuses = ['backlog', 'discovery', 'alpha', 'beta', 'live']
-    const themes = formatRawThemeData(rawThemes)
-    console.log('GET index')
+    const rawProducts = await productService.getEntries('product');
+    const rawThemes = await productService.getEntries('themes');
```
Comment (jonhindle) [inline server/routes/index.js:9]: Maybe not for now, but I think these could be fetched in parallel with 'Promise.all'.

File: server/services/products.js [hunk 5]
```diff
@@ -1,26 +1,36 @@
...
       const entries = await contentfulClient.getEntries({
-        content_type: contentType
-      })
-      if (entries.items) return entries.items
-      console.log(`Error getting Entries for ${contentType}.`)
+        content_type: contentType,
+      });
+      if (entries.items) {
+        return entries.items;
+      }
```
Comment (jonhindle) [inline server/services/products.js:10]: Could we destruct here too?
```
const { items } = await contentfulClient.getEntries({
   content_type: contentType
});
if (items) return items;
```

File: server/routes/index.js (hunk 4 above)
Comment (psoleckimoj) [reply server/routes/index.js:9]: I haven't touched this code.

File: server/services/products.js (hunk 5 above)
Comment (psoleckimoj) [reply server/services/products.js:10]: Also haven;'t touched code.

**Review by jonhindle — APPROVED** [review]

**Pull request merged**